lightly/active_learning/scorers/classification.py

```diff
@@ -27,18 +27,14 @@ def _entropy(probs: np.ndarray, axis: int = 1) -> np.ndarray:
 
 
 class ScorerClassification(Scorer):
-    """A class to compute active learning scores out of the model_output (i.e. the predictions of a model) for a classification task.
```

Comment:
I do not get the point of all the changes in the docs of this class. 
'Helper class' - There is no widespread definition of a helper class thus it does not provide any informations
'i.e. predictions ... classification task' - I think it should be specified that this class is only targeted at classification tasks.
change of description of model_output - in my eyes the previous doc is much more informative and the relevant information is easier to find. I also think that we can include some pseudocode here, as it helps to understand how to generate the desired format.

Reply:
I agree to a certain degree but there were a few problems with your docstrings:
- Line 30 is way too long - that's why I changed it. The "helper" is not necessary.
- The model_output part was formatted in a way the documentation could not display.

I will rework the part about the model_output to include all the information it had before and add an example.

Reply:
Ok, so the previous docs only looked well-formatted in python, but badly-formatted in the documentation. Yes, this is a very good reason to change the docs.